roles/cifmw_external_dns/tasks/cert.yml

```diff
@@ -0,0 +1,83 @@
+---
+# Copyright 2024 Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+- name: Put the certificate manifest file in place
+  delegate_to: localhost
+  ansible.builtin.template:
+    src: "templates/cert.yml.j2"
+    dest: "{{ cifmw_external_dns_manifests_dir }}/{{ cifmw_external_dns_cert_name }}.yml"
+    mode: "0644"
+    force: true
+
+- name: Create certificate
+  when: not (cifmw_external_dns_check_mode | default (false) | bool)
+  block:
+    - name: Apply k8s manifest file of kind Certficate
+      delegate_to: localhost
```

Comment:
same - if we generate one resource per compute/ceph, fine, else care to add `run_once: true` ?